utils.css

```diff
@@ -1,24 +1,31 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400&display=swap');
 :root{
     --primary-color:#3182ce;
     --blue:#2b6cb0;
-    --red:rgb(255 0 0);
-    --grey:grey;
+    --red:rgb(255, 0, 0);
+    --grey:#646464cf;
     --ltblue:#0084ff;
     --green1:#12b2b3;
     --ltgreen:#04bf5b;
     --blue1:rgb(2, 177, 200);
```

Comment:
For naming, CSS color variables it will be good if you will use `--COLOR--BLUE-1` syntax. As it improves the readability.